init.lua

```diff
@@ -233,6 +234,20 @@ local function openUrlInNewTab(url)
   hs.osascript.applescript(script)
 end
 
+local function openUrlInNewBackgroundTab(url)
+  local script = [[
+      tell application "Safari"
+        activate
+        tell window 1
+          make new tab with properties {URL:"%s"}
+        end tell
+      end tell
+      return
```

Comment:
This `return` is needed so that the script creates no output (just like the `openUrlInNewTab` script).

Without this `return` the script will output something like `tab 6 of window id 82974` which seems to halt the spoon.